README.md

```diff
@@ -2,3 +2,46 @@
 
 [![Build Status](https://travis-ci.org/alrra/travis-scripts.svg?branch=master)](https://travis-ci.org/alrra/travis-scripts)
 [![devDependency Status](https://david-dm.org/alrra/travis-scripts/dev-status.svg)](https://david-dm.org/alrra/travis-scripts#info=devDependencies)
+
+## How to use
+
+1. Save `travis-scripts` as a `devDependency` to `package.json`:
+
+    ```sh
+    $ npm install travis-scripts --save-dev
```

Comment:
Fixed in https://github.com/alrra/travis-scripts/commit/e11dabc3a5a7371fbe8d1bf6ae70a094b55c0212.